include/depthai/openvino/OpenVINO.hpp

```diff
@@ -19,6 +19,8 @@ class OpenVINO {
     /// OpenVINO Version supported version information
     enum Version { VERSION_2020_3, VERSION_2020_4, VERSION_2021_1, VERSION_2021_2, VERSION_2021_3, VERSION_2021_4, VERSION_2022_1 };
 
+    // Device for which the blob is compiled
+    enum Device { VPU, VPUX };
```

Comment:
```suggestion
    enum class Device { VPU, VPUX };
```

As it then reads: `OpenVINO::Device::VPU / VPUX` instead of `OpenVINO::VPU / VPUX`

Reply:
Version was done as regular enum intentional as name includes the `VERSION_` prefix, as number otherwise isn't a valid starting character in enum name

Reply:
Makes sense yes, I will fix those.